test/cucumber/ResidueGeneratorProductionReport.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Residue Generator Production Report
+  As a researcher
+  I want visualize the waste production of the residue generators
+  So I can collect data for my research
+
+  @ignore
+  Scenario: generate month based Residue Production report
+  Given the system has a residue generator at "endereco" registered
+  When the system generates an production report for the last "5" months
+  Then the residue generator at "endereco" appears at the report
+
+  @ignore
+  Scenario: generate empty month based Residue Production report
```

Comment:
do ponto de vista do cenário, escreva um passo que represente isso. algo como "Then the lists of productions and generators remain the same". do ponto de vista de implementação desse passo, teria que comparar o conteúdo atuais das listas com conteúdos coletados em um passo anterior do teste.